src/rider/main/kotlin/com/jetbrains/rider/plugins/unreal/actions/PlayActions.kt

```diff
@@ -198,4 +202,21 @@ class SingleStepInUnrealAction : PlayStateAction(
         state.disableUntilStateChange()
         host.model.requestFrameSkipFromRider.fire(state.nextRequestID())
     }
+}
+
+class RefreshProjects : DumbAwareAction(AllIcons.Actions.Refresh) {
+    override fun update(e: AnActionEvent) {
+        super.update(e)
+        val host = e.getHost()
+        e.presentation.isVisible = host?.isUnrealEngineSolution ?: false
```

Comment:
I suggest to be more safe here and set not only visible to false, but also disable action.
AFAIK invisible but enabled actions are still present in search-anywhere and could be triggered by hotkey.